docs/modules/general/pages/architecture-overview/index.adoc

```diff
@@ -1,20 +1,26 @@
 = OneCX Architecture Overview
 
+:idprefix:
+:idseparator: -
+
 :workspace_url: xref:latest@onecx-workspace:general:index.adoc
 :product-store_url: xref:latest@onecx-product-store:general:index.adoc
```

Comment:
what is the naming convention/separator for these:
- product-store_url
- product-store-url
- product_store_url